src/celengine/octree.h

```diff
@@ -300,9 +300,9 @@ inline void DynamicOctree<OBJ, PREC>::rebuildAndSort(StaticOctree<OBJ, PREC>*& _
     OBJ* _firstObject = _sortedObjects;
 
     if (_objects != nullptr)
-        for (typename ObjectList::const_iterator iter = _objects->begin(); iter != _objects->end(); ++iter)
+        for (typename ObjectList::iterator iter = _objects->begin(); iter != _objects->end(); ++iter)
```

Comment:
could you rewrite this using C++11 style?

Reply:
I got
> src/celengine/dsodb.cpp:385:46:   required from here
> src/celengine/octree.h:303:9: error: no matching function for call to ‘begin(std::vector<DeepSkyObject**, std::allocator<DeepSkyObject**> >*&)’

Reply:
for `for (const auto &o : *_objects)`?

Reply:
Ah, forgot that `_objects` is a pointer...